service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -396,10 +396,18 @@ public ResponseEntity<ApiCloneWorkspaceResult> cloneWorkspace(
     Optional<SpendProfileId> spendProfileId =
         Optional.ofNullable(body.getSpendProfile()).map(SpendProfileId::new);
     final UUID destinationWorkspaceId = UUID.randomUUID();
+
+    // ET uses userFacingId; CWB doesn't. Schema enforces that userFacingId must be set. CWB doesn't
+    // pass userFacingId in request, so use id. Prefix with "a" because userFacingId must start with
+    // letter.
+    String destinationUserFacingId = Optional.ofNullable(body.getUserFacingId()).orElse("a" + destinationWorkspaceId);
```

Comment:
nit: a hyphen after the 'a' would make it more obvious that it's a name and not a guid in log messages and other contexts where it's maybe confusable.

Reply:
Done. Updated clone and create.